Modules/Synchronization/src/Synchronization.Application/SyncRuns/Commands/DeleteSyncRunsByIdentity/Handler.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Backbone.Modules.Synchronization.Application.Infrastructure;
+using Backbone.Modules.Synchronization.Domain.Entities;
+using Backbone.Modules.Synchronization.Domain.Entities.Sync;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backbone.Modules.Synchronization.Application.SyncRuns.Commands.DeleteSyncRunsByIdentity;
+public class Handler(ISynchronizationDbContext dbContext) : IRequestHandler<DeleteSyncRunsByIdentityCommand>
+{
+    public async Task Handle(DeleteSyncRunsByIdentityCommand request, CancellationToken cancellationToken)
+    {
+        await dbContext.Set<Datawallet>().Where(d => d.Owner == request.IdentityAddress).ExecuteDeleteAsync(cancellationToken);
```

Comment:
Isn't this the responsability of the `DeleteDatawalletsByIdentityCommand`?

Reply:
Yes, this was duplicated. Ty